endpoints/api/logs.py

```diff
@@ -103,14 +104,17 @@ def _get_aggregate_logs(
     (start_time, end_time) = _validate_logs_arguments(start_time, end_time)
```

Comment:
we could fail early by adding validation logic in this function instead of down the code